Gemfile

```diff
@@ -7,7 +7,7 @@ group :test do
   gem 'berkshelf'
   gem 'test-kitchen'
   gem 'kitchen-vagrant'
-  gem 'chefspec'
+  gem 'chefspec', '~> 4'
```

Comment:
Latest ChefSpec is 7.x, I added `'~> 4'` to ensure the latest 4.x is used.  I didn't go down the Chef 13 support path, 😬